docs/env.md

```
@@ -1,45 +1,58 @@
 # BE Environment Variables
 
-## Required
-- `SPRING_PROFILES_ACTIVE`: `dev` / `stage` / `prod`
-- `DB_URL`: PostgreSQL JDBC URL
-- `DB_USERNAME`: DB user
-- `DB_PASSWORD`: DB password
-- `JWT_SECRET`: JWT signing secret (at least 32 bytes)
+## 필수 (Required)
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)